src/gui/tracks/TrackView.cpp

```diff
@@ -451,6 +440,16 @@ void TrackView::muteChanged()
 }
 
 
+void TrackView::onTrackGripGrabbed()
+{
+	m_action = Action::Move;
+}
+
+void TrackView::onTrackGripReleased()
+{
+	m_action = Action::None;
+}
```

Comment:
I think it's best to define simple functions like these within the header?

Reply:
IMO it's better to keep them in the implementation file because it is an implementation detail and a potentially complex operation that's triggered due to a signal/slot connection. Even if it is not complex in this case.